liquibase_changelog/patternatlas.xml

```diff
@@ -3,6 +3,6 @@
                    xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
                    xmlns:pro="http://www.liquibase.org/xml/ns/pro" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
                    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/pro http://www.liquibase.org/xml/ns/pro/liquibase-pro-4.6.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.6.xsd">
-    <include file="file:patternatlas_schema.xml"/>
-    <include file="file:patternatlas_constraints.xml"/>
+    <include file="patternatlas_schema.xml"/>
```

Comment:
please change the includes back to "file:..." again. The docker version used in deployment might not work without it.